src/openrct2/ride/CarEntry.h

```diff
@@ -202,7 +203,7 @@ struct CarEntry
     uint8_t sprite_height_negative;
     uint8_t sprite_height_positive;
     CarEntryAnimation animation;
-    uint32_t flags;
+    uint64_t flags;
```

Comment:
Just to check: does this require a park version bump?

Reply:
It doesn’t, but it’s good you checked.